static/src/js/util/collectionMetadata/spatial.js

```diff
@@ -38,8 +36,8 @@ export const buildSpatial = (json) => {
 
         spatialList.push(`Bounding Rectangle: (${degrees(north)}, ${degrees(west)}, ${degrees(south)}, ${degrees(east)})`)
       })
-    } else if (geometry.gPolygons) {
-      const polygons = castArray(geometry.gPolygons)
+    } else if (geometry.gpolygons) {
```

Comment:
There are issues with this in prod. Graphql is returning these values as camelcase lowered so if it is a polygon or a line EDSC is defaulting to 'not available' for the label